incubator/elasticsearch-curator/templates/cronjob.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: batch/v1beta1
```

Comment:
don't hardcode the apiVersion here, see these example [helper](https://github.com/kubernetes/charts/blob/master/incubator/jaeger/templates/_helpers.tpl#L3-L12) and [how to apply it](https://github.com/kubernetes/charts/blob/ae2a474b0c832edc92345bd87e48130c50ced50c/incubator/jaeger/templates/spark-cronjob.yaml#L2-L3)